tests/unit/views/pacbio/PacbioPoolCreate.spec.js

```diff
@@ -110,7 +89,10 @@ describe('PacbioPoolCreate', () => {
       }
       mockFetchPacbioTagSets.mockReturnValue({ success: true, errors: [] })
       mockPopulateUsedAliquotsFromPool.mockReturnValue({ success: true, errors: [] })
-      const { wrapperObj, storeObj } = mountWithStore({ plugins, state })
+      const { wrapperObj, storeObj } = mountPacbioPoolCreate({
+        plugins,
+        initialState: { pacbioPoolCreate: state },
+      })
       wrapper = wrapperObj
```

Comment:
similar thing here. If we are constantly reassigning wrapperObj and storeObj to wrapper and store would it make sense to either call them wrapper and store or wrapperObj and storeObj.